content/workshops/rhel_8/exercise1.0.adoc

```diff
@@ -0,0 +1,236 @@
+---
+title: "Exercise 1.0 - Welcome to RHEL!"
+workshops: rhel_8
+workshop_weight: 10
+layout: lab
+---
+
+:domain_name: redhatgov.io
+:icons: font
+:imagesdir: /workshops/rhel_8/images
+
+
+= Exercise 1.0 - Welcome to RHEL!
+
+
+== Exercise Description
+
+Welcom to RHEL!
+
+Red Hat Enterprise Linux 8 is the intelligent enterprise Linux, providing a secure and consistent foundation across hybrid cloud deployments and the tools enterprises need to deliver services and workloads faster with less effort.  RHEL 8 enables delivery of any application, on any footprint, at any time.
+
+With predictable release cadence and 10-year enterprise support, RHEL 8 is the safe and stable choice for your enterprise.
+
+RHEL 8 also automates regulatory compliance and security configuration remediation ​across your system and within containers​​ with OpenSCAP, Red Hat’s National Institute of Standards and Technology (NIST)-certified scanner.  RHEL 8's security portfolio also includes a new, system-wide encryption policy and the latest nftables/firewalld firewall.
```

Comment:
Line 24: Consider - RHEL 8 uses OpenSCAP on your system, to automate regulatory compliance and security configuration. OpenSCAP is the National Institute of Standards and Technology (NIST)-certified scanner, used by Red Hat.